usb-autorunner/debian/patches/series

```diff
@@ -1,2 +1,3 @@
 add-service-scripts
 Update-autorunner-script
+fix-autorunner-script-path
```

Comment:
And this file?

Reply:
This is the list of all available patches